develop-docs/sdk/telemetry/spans/span-api.mdx

```diff
@@ -140,4 +161,23 @@ unrelatedSpan.end();
 on('checkout-finished', ({ timestamp }) => {
 	checkoutSpan.end(timestamp);
 })
-```
\ No newline at end of file
+```
+
+```Python {tabTitle:Python}
+
+with sentry_sdk.start_span(name="checkout", attributes={"user.id": "123"}) as checkout_span:
```

Comment:
Q: Can `sentry_sdk.start_span` be used without a `with` statement? 

If so, could we write down an example for this? I think it's fully okay to support `with` (or callbacks or whatever the platform prefers) but ideally we can also document the most simple case which is that `start_span` returns an active span which users end manually.